### PR TITLE
Backport compression 1.4.6

### PR DESCRIPTION
**Pull request by yuval-k**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/3663

### CODE REVIEW

File: .github/workflows/regression-test.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         go-version: 1.14
       id: go
     - name: Install Protoc
-      uses: arduino/setup-protoc@v1.1.0
+      uses: solo-io/setup-protoc@master
```
Comment (kdorosh) [inline .github/workflows/regression-test.yaml:20]: let's pin to a version

Comment (yuval-k) [reply .github/workflows/regression-test.yaml:20]: i agree; do note i copied it from master so we may want to do the same change there

Comment (yuval-k) [reply .github/workflows/regression-test.yaml:20]: just check, there's not a version to pin to..

**Review by kdorosh — APPROVED** [review]

**Pull request merged**